javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj) {
+  let nameTargeted = obj.name;
+  let ageTargeted = obj.age;
+  return [`<li>name: ${nameTargeted}</li>`, `<li>age: ${ageTargeted}</li>`];
```

Comment:
- Good work on these so far!
- For this one, the challenge wants us to return an arr of **key value** pairs. On line 22, 23 double check that you are grabbing the key AND the value and returning both as html elements in the arr. 👍  (You worked with object.entries() before...that may help here).